examples/guestbook/README.md

```diff
@@ -31,31 +37,31 @@ Use the file `examples/guestbook/redis-master.json` which describes a single pod
         "image": "dockerfile/redis",
         "cpu": 100,
         "ports": [{
-          "containerPort": 6379,
-          "hostPort": 6379
+          "containerPort": 6379,   # containerPort: Where traffic to redis ultimately is routed to.
+          "hostPort": 6379         # hostPort: The port on the host itself, that the service proxy sends traffic to.
         }]
       }]
     }
   },
   "labels": {
-    "name": "redis-master"
+    "name": "redis-master" # This label needed for when we start our redis-master service.
   }
 }
 ```
 
-Create the redis pod in your Kubernetes cluster by running:
+Now, create the redis pod in your Kubernetes cluster by running:
 
 ```shell
-$ cluster/kubectl.sh create -f examples/guestbook/redis-master.json
+kubectl create -f examples/guestbook/redis-master.json
 ```
 
 Once that's up you can list the pods in the cluster, to verify that the master is running:
 
 ```shell
-cluster/kubectl.sh get pods
+kubectl get pods,services,rc
```

Comment:
Why are we listing services and replication controllers ? User has not yet interacted with them (to this point), hence listing them doesn't provide a benefit for the user IMHO.